lib/rspec-puppet-facts.rb

```diff
@@ -379,9 +371,6 @@ def self.facter_version_to_loose_requirement_string(version)
     elsif /\A[0-9]+\Z/.match?(version)
       # Interpret 3 as < 4
       "< #{version.to_i + 1}"
-    else
-      # This would be the same as the strict requirement
```

Comment:
I originally wrote this because it explicitly documents the flow. Is it worth keeping that?